docs/_js/examples/todo.js

```diff
@@ -3,18 +3,21 @@ class TodoApp extends React.Component {
   constructor(props) {
     super(props);
     this.handleChange = this.handleChange.bind(this);
+    this.handleReset = this.handleReset.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
-    this.state = {items: [], text: ''};
+    this.originalValues = {items: [], text: ''};
+    this.state = this.originalValues;
```

Comment:
Just curious, why {items: [], text: ''} is first assigned to this.originalValues and then this.state
Why this change was made ?